contrib/objcwrapper/header_static/makefile

```diff
@@ -0,0 +1,7 @@
+bin/header_static:
+	mkdir -p bin
+	../../../bin/nitc --dir bin src/header_static.nit
+
+tests: bin/header_static
+	cat CGGeometry.h | bin/header_static CGGeometry.h > static_CGGeometry.h
+	cat NSObject.h | bin/header_static NSObject.h > static_NSObject.h
```

Comment:
In the `cat`, you can use the full path to the source files, even if they are only on OS X. It will make testing easier.